src/neo_omnidrive_socketcan.cpp

```diff
@@ -151,6 +151,13 @@ class NeoSocketCanNode {
 		m_node_handle.param("measure_torque", m_measure_torque, false);
 		m_node_handle.param("homeing_button", m_homeing_button, 0);
 		m_node_handle.param("diagnostics_prefix", m_diagnostics_prefix, std::string(""));
+		// these values are used to calculate and store max_accel_enc_s for each motor
+		double wheel_radius;
+		if(!m_node_handle.getParam("wheel_radius", wheel_radius)) {
+			throw std::logic_error("missing wheel_radius param");
+		}
+		double max_drive_acc;
+		m_node_handle.param("max_drive_acc", max_drive_acc, 0.0);  // in m/s^2
```

Comment:
I believe you wanted to say `// in ticks/s^2`

Reply:
The value is indeed in m/s^2; I thought that a more human-readable version in SI units would be preferable.

Down in line 216 you can see the conversion from m/s^2 to ticks/s^2 - this is also the reason why I need to read the radius parameter.

I have prepared an alternative version of this PR where instead the acceleration is specified in ticks/s^2 for each drive motor individually; your choice :)